### PR TITLE
Ensure 'ZOPE2PATH' is text, not bytes

### LINKED ISSUE #372: mkwsgiinstance:  Fails under Py3k on Unix filesystem
In a fresh Zope 4.0b6 install:

```bash
$ bin/mkwsgiinstance -d .
Please choose a username and password for the initial user.
These will be the credentials you use to initially manage
your new Zope instance.

Username: xxx
Password: 
Verify password: 
Traceback (most recent call last):
  File "bin/mkwsgiinstance", line 11, in <module>
    sys.exit(main())
  File "/tmp/zope4/lib/python3.6/site-packages/Zope2/utilities/mkwsgiinstance.py", line 142, in main
    copyzopeskel.copyskel(skelsrc, skeltarget, None, None, **kw)
  File "/tmp/zope4/lib/python3.6/site-packages/Zope2/utilities/copyzopeskel.py", line 182, in copyskel
    copydir(root, dirs, files, targetdir, replacements, uid, gid)
  File "/tmp/zope4/lib/python3.6/site-packages/Zope2/utilities/copyzopeskel.py", line 206, in copydir
    gid)
  File "/tmp/zope4/lib/python3.6/site-packages/Zope2/utilities/copyzopeskel.py", line 225, in copyin
    text = text.replace("<<%s>>" % k, replacements[k])
TypeError: replace() argument 2 must be str, not bytes
```

Debugging, the issue is that `ZOPE2PATH` is bytes.

### PR DESCRIPTION
**Pull request by tseaver**
Closes #372.

### CODE REVIEW

File: src/Zope2/utilities/mkwsgiinstance.py [hunk 1]
```diff
@@ -218,7 +218,7 @@ def get_zope2path(python):
         output = subprocess.check_output(
             [python, '-c', 'import Zope2; print(Zope2.__file__)'],
             stderr=subprocess.PIPE)
-        zope2file = output.strip()
+        zope2file = output.strip().decode('utf-8')
```
Comment (davisagli) [inline src/Zope2/utilities/mkwsgiinstance.py]: I think it should be decoded for Python 3 only; the return value from this function ends up getting used in a str.replace on the content of files opened in text mode.

Comment (tseaver) [reply src/Zope2/utilities/mkwsgiinstance.py]: `str.decode` returns `unicode` on Python2.x, which is what we want, I believe.

Comment (davisagli) [reply src/Zope2/utilities/mkwsgiinstance.py]: I don't think so. The line that broke in Python 3 in #372 was:

    text = text.replace("<<%s>>" % k, replacements[k])

where `replacements[k]` comes from the `zope2file` value.

`text` is a `str` so we want `zope2file` to also be a `str`, which means not decoding it in Python 2.

**Comment by icemac** [comment]
Superseded by #371.

**Pull request closed without merging**